src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -165,6 +165,33 @@
   (let [price (get-market-value currency token)]
     (money/crypto->fiat balance price)))
 
+(defn formatted-token-fiat-value
+  "Converts a token balance into its equivalent fiat value, formatted with a currency symbol.
+  If the fiat value is below $0.01, it returns a <$0.01 string"
+  [{:keys [currency currency-symbol balance token]}]
+  (let [price             (or (get-market-value currency token) 0)
+        price-zero?       (zero? price)
+        balance           (or balance 0)
+        balance-positive? (pos? balance)
+        fiat-value        (money/crypto->fiat balance price)
+        fiat-value-zero?  (money/equal-to fiat-value (money/bignumber 0))]
+    (if (and fiat-value-zero? (not price-zero?) balance-positive?)
+      (number/small-number-threshold 2 currency-symbol)
+      (str currency-symbol fiat-value))))
+
+(defn sanitized-token-amount-to-display
+  "Formats a token amount to a specified number of decimals.
+  Returns a threshold value if the formatted amount is less than the minimum value represented by the decimals."
+  [amount display-decimals]
+  (let [number                (or (money/bignumber amount)
+                                  (money/bignumber 0))
+        amount-fixed-decimals (number/to-fixed number display-decimals)]
+    (if (and (= amount-fixed-decimals "0")
```

Comment:
why "0" and not `0`?

Reply:
That's because `number/to-fixed` calls `remove-trailing-zeroes` fn which returns a string